requests_unixsocket/adapters.py

```diff
@@ -39,9 +39,9 @@ def connect(self):
 
 class UnixHTTPConnectionPool(urllib3.connectionpool.HTTPConnectionPool):
 
-    def __init__(self, socket_path, timeout=60):
+    def __init__(self, socket_path, timeout=60, maxsize=1, **kwargs):
```

Comment:
The default value upstream is `10`: https://github.com/docker/docker-py/commit/be7d0f01844d5c08ee157446ce96f5bc6381507c#diff-25eefff144d019c2dcb7e072a6a380825e67c192831eef3d59ca5ea2ca65aa17.

@mupuf should we mimic that?

Reply:
Ideally, the original version should have mirrored the upstream default... but this ship has sailed now and we don't want to regress on someone's code just for purity.

Let's start an issue about things that we want to fix for the next major version? There we could address your comment above about redefining private method.